backend/stream.go

```diff
@@ -34,7 +34,7 @@ type StreamHandler interface {
 type SubscribeStreamRequest struct {
 	PluginContext PluginContext
 	Path          string
-	Query         json.RawMessage
```

Comment:
This is a breaking change in the sdk which should be fine given the readme of the repo

Reply:
Hi, that's a rename that targets `query_for_subscribe_and_runstream` branch - which is not merged yet. Sorry - had to emphasise this better in the description.

Reply:
Ohh sorry